src/rest/util.ts

```diff
@@ -82,13 +82,30 @@ export const isAuthenticated = async (
 	return true;
 };
 
+/**
+ * Used to convert what is supposed to be a numerical representation of a boolean value
+ * since the request body sends everything as a string
+ * Returns 0, 1, or null
+ * @param val
+ * @returns numerical representation of a boolean value or null
+ */
+export const convertToNum = (val: string): 0 | 1 | null => {
+	if (val === "0") {
+		return 0;
+	} else if (val === "1") {
+		return 1;
+	} else {
+		return null;
+	}
+};
+
```

Comment:
At the time of this function's creation, I could not think of a better solution for converting the string 'bool' values to actual nums, so please please lmk if you have any suggestions to make this better!!

Reply:
how about:
``ts
const num = Number(val);
return num === NaN ? null : num;
``